Update RELEASING.md.template

stefannibrasil opened the pull request:
Include changelog and mention RubyGem Trusted Publication method, and formats the content.

stefannibrasil committed:
Update RELEASING.md.template

Include changelog and mention RubyGem Trusted Publication method

louis-antonopoulos commented on RELEASING.md.template:
````diff
@@ -14,10 +14,14 @@
     gem push project-name-*.gem
     ```
-1. Add a new GitHub release using the recent `NEWS.md` as the content. Sample
+1. Add a new GitHub release using the recent `NEWS.md` or `CHANGELOG.md` as the content. Sample
````
Not directly related to this PR, but I found the line break between `Sample` and `URL` to be confusing when I was working recently on yuri-ita (which used this template).

What would you think of moving `URL` from the next line onto this one?

louis-antonopoulos replied:
Addding a bit more thought -- I found the line breaks to be confusing more broadly. Would you be up for consolidating shorter lines into longer ones, too?

stefannibrasil replied:
oh, you're right. What do you think of [f026bc4](https://github.com/thoughtbot/templates/pull/39/commits/f026bc4ff395341040786a0b81369f50d6b9a625) ? I might have gone too far, but this file had lots in it 👀

louis-antonopoulos replied:
I love the changes!

stefannibrasil committed:
Better formatting for this file

@louis-antonopoulos mentioned the line breaks weren't
helpful, so suggesting breaking the steps into
sections to help with readibility.

louis-antonopoulos commented on RELEASING.md.template:
```diff
@@ -1,27 +1,43 @@
+
+## 3. Commit changes
+
+There shouldn't be code changes, and thus CI doesn't need to run. You can then add "[ci skip]" to the commit message.
```
Shortening this a little to make it clear that the reader should do this (and not have to think about if they should because of the "you can"):

```
There shouldn't be code changes, and thus CI doesn't need to run. Add "[ci skip]" to the commit message.
```

stefannibrasil replied:
👍🏼

louis-antonopoulos commented on RELEASING.md.template:
```diff
@@ -1,27 +1,43 @@
 If the project is hosted on RubyGems, consider using [RubyGems Trusted Publication] to automatically
 push the release using a GitHub action.
 
+## 7. Add a new GitHub release using the recent `NEWS.md` or `CHANGELOG.md` as the content.
```
[minor]
The trailing `.` could be ommitted here to be consistent with the other steps.

stefannibrasil replied:
👀

louis-antonopoulos commented on RELEASING.md.template:
```diff
@@ -1,27 +1,43 @@
+Make sure to say "thank you" to the contributors who helped shape this version!
+
 [RubyGems Trusted Publication]: https://github.com/rubygems/release-gem
 [_quick guide on how to sign a release_]: http://gitready.com/advanced/2014/11/02/gpg-sign-releases.html
```
Should this line reference an `https` link?

stefannibrasil replied:
👍🏼

louis-antonopoulos reviewed (approved):
None of my comments are blocking. Thanks for reworking the shape of this file! It feels much more readable to me.

stefannibrasil committed:
Great suggestions from Louis :)

stefannibrasil commented:
I love your suggestions, thank you, Louis! Addressed your suggestions in [e734581](https://github.com/thoughtbot/templates/pull/39/commits/e734581802369274edfd2cbd789b7ee92a854a18)
